changelog/18.0/18.0.0/release_notes.md

```diff
@@ -0,0 +1,303 @@
+We have realized now that there are cases when the replication is broken but all the tablets are reachable. In these cases, it is advisable to 
+call `EmergencyReparentShard` with `--wait-for-all-tablets` so that it doesn't ignore one of the tablets.
+
+#### <a id="new-vtgate-streaming-sesion"/>VTGate GRPC stream execute session flag `--grpc-send-session-in-streaming`
+
+This flag enables transaction support on `StreamExecute` api.
+One enabled, VTGate `StreamExecute` grpc api will send session as the last packet in the response.
+The client should enable it only when they have made the required changes to expect such a packet.
+
+It is disabled by default.
+
+### <a id="foreign-keys"/>Experimental Foreign Key Support
+
+A new field `foreignKeyMode` has been added to the Vschema. This field can be provided for each keyspace. The Vtgate flag `--foreign_key_mode` has been deprecated in favour of this field.
+
+There are 3 foreign key modes now supported in Vitess -
+1. `FK_UNMANAGED` -
+   This mode represents the default behaviour in Vitess, where it does not manage foreign keys column references. Users are responsible for configuring foreign keys in MySQL in such a way that related rows, as determined by foreign keys, reside within the same shard.
+2. `FK_MANAGED` [EXPERIMENTAL] -
+   In this experimental mode, Vitess is fully aware of foreign key relationships and actively tracks foreign key constraints using the schema tracker. Vitess takes charge of handling DML operations with foreign keys cascading updates, deletes and verifying restrict. It will also validate parent row existence.
+   This ensures that all the operations are logged in binary logs, unlike MySQL implementation of foreign keys.
+   This enables seamless integration of VReplication with foreign keys.
+   For more details on what operations Vitess takes please refer to the [design document for foreign keys](https://github.com/vitessio/vitess/issues/12967).
+3. `FK_DISALLOW` -
+   In this mode Vitess explicitly disallows any DDL statements that try to create a foreign key constraint. This mode is equivalent to running Vtgates with the flag `--foreign_key_mode=disallow`.
+
+#### Upgrade process
+
+After upgrading from v17 to v18, the users should specify the correct foreign key mode for all their keyspaces in the Vschema using the new property. Once this change has taken effect, the deprecated flag `--foreign_key_mode` can be dropped from all the Vtgates.
+
+### <a id="vtadmin"/>VTAdmin
+
+#### <a id="updated-node"/>vtadmin-web updated to node v18.16.0 (LTS)
+
+Building vtadmin-web now requires node >= v18.16.0 (LTS). Breaking changes from v16 to v18 are listed
+in https://nodejs.org/en/blog/release/v18.0.0, but none apply to VTAdmin. Full details on v18.16.0 are listed
+here https://nodejs.org/en/blog/release/v18.16.0.
+
+### <a id="deprecations-and-deletions"/>Deprecations and Deletions
+
+#### <a id="deprecated-flags"/>Deprecated Command Line Flags
+
```

Comment:
All the flags listed in this section begin with: `--.... is deprecated and will be removed ...`. I do not have strong opinion, but it feels redundant, we can probably drop the `is deprecated` part as we are already in the `Deprecated Command Line Flags` section.